pkg/agents/team2/policies.go

```diff
@@ -41,3 +55,47 @@ func (a *CustomAgent2) adjustPolicies() {
 		}
 	}
 }
+
+func (a *CustomAgent2) winOrLose(state int) float64 {
+	sumPolicies := 0.0
+	sumAveragePolicies := 0.0
+	Delta_win := 0.025
+	Delta_lose := 0.1
+	for i := 0; i < len(a.actionSpace); i++ {
+		sumPolicies += a.policies[state][i] * a.qTable[state][i]
+		sumAveragePolicies += a.averagePolicies[state][i] * a.qTable[state][i]
+	}
+	if sumPolicies > sumAveragePolicies {
+		return Delta_win
+	}
+	return Delta_lose
+}
+
+func (a *CustomAgent2) exportPolicies() {
+	f, err := os.OpenFile(fmt.Sprintf("%s%s%s", a.ID(), "policies", ".csv"), os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0600)
+	if err != nil {
+		panic(err)
```

Comment:
Do not use panic. Throw an error if you're going to do this, but don't use panic.

Reply:
Fixed